docs/getting-started/rulesets.md

```diff
@@ -199,3 +198,141 @@ rules:
 The example above will run the recommended rules from the `spectral:oas2` ruleset, but report `operation-2xx-response` as a warning rather than as an error (as is the default behavior in the `spectral:oas2` ruleset).
 
 Available severity levels are `error`, `warn`, `info`, `hint`, and `off`.
+
+
+### Adding a custom function
+
+If the built-in functions are not enough, Spectral gives you a possibility of providing your own ones.
+Previously, this used to be possible only in case of programmatic usage of Spectral, but there is no such limitation anymore,
+and functions are an integral part of Spectral rulesets.
+
+A custom function might be any JS function compliant with [IFunction](https://github.com/stoplightio/spectral/blob/90a0864863fa232bf367a26dace61fd9f93198db/src/types/function.ts#L3#L8) type.
```

Comment:
Suggest linking to where the file will be once this is merged - otherwise will have to open another PR to fix the link.

Reply:
This is unchanged, though. I didn't touch `IFunction`, it's already on master.

Reply:
@P0lip Marc is suggesting that a link to it on master would be better. I'd agree, so long as we don't change it a bunch or move it.